juniper_official/routing/check-bgp-neighbor-stats.rule

```diff
@@ -168,8 +168,8 @@
                     juniper {
                         operating-system junos {
                             products EX {
-                                platforms All {
```

Comment:
Add sensor at product level

Reply:
Added sensor at EX Product level for both the rules.